src/utils/components/CPUMemory/CPUMemory.tsx

```diff
@@ -19,17 +22,15 @@ const CPUMemory: FC<CPUMemoryProps> = ({ vm }) => {
   const { t } = useKubevirtTranslation();
   const itMatcher: V1InstancetypeMatcher = vm?.spec?.instancetype;
   const { instanceType, instanceTypeLoaded, instanceTypeLoadError } = useInstanceType(itMatcher);
+  const isVMRunning = vm?.status?.printableStatus === printableVMStatus.Running;
   const { vmi, vmiLoadError } = useVMI(getName(vm), getNamespace(vm));
```

Comment:
u are doing 2 network calls for vmi with your ned hook, we dont want this. u can return vmi from your hook.

Reply:
Where ate those network calls? I cannot see it. Could you be more specific? Thanks.

Reply:
useVMI, useCPUData both call vmi - its extra call which is uneeded